src/api/v1/users/password.ts

```diff
@@ -7,15 +7,15 @@ import {
   email as emailSchema,
   password as passwordSchema,
 } from './schema';
-import { EmailTemplate } from '../../../services/email/templates';
+import { Templates } from '../../../services/email';
 import { BoomWithValidation } from '../utils';
-import { AppEnum } from '../types';
+import { AppEnum } from '../../../types/internal';
```

Comment:
Do you think it would be easier for us to setup a types folder in root? I feel like I often spend time wondering where the types are for various bits n pieces.

Reply:
Yeah, TBH I wanted a way to make all our type definitions available globally (since they are ephemeral anyway, it doesn't matter), but could never figure out a way to do it.

One big root-level types file/folder might make sense. Not in this PR though :)